data/tutorials/language/0it_00_values_functions.md

```diff
@@ -593,6 +606,20 @@ It looks like two arguments have been passed: `"hello"` and `"world"`. However,
 
 In many imperative languages, the `spicy_cat ("hello", "world")` syntax reads as a function call with two parameters; but in OCaml, it denotes applying the function `spicy_cat` to a tuple containing `"hello"` and `"world"`.
 
+### Records as Function Paramters
+
+It is possible to use the record syntax to specify function parameters. Here is how it can be used to define yet another version of the running example:
+```ocaml
+# type cat_args = {x : string; y : string};;
+type cat_args = { x : string; y : string; }
+
+# let happy_cat ({x; y}: cat_args) = x ^ " " ^ y;;
+val happy_cat : cat_args -> string = <fun>
+
+# happy_cat {x = "hello"; y = "world"};;
+- : string = "hello world
+```
+
```

Comment:
Adding this section is a good idea. I propose
- Move it right after "pattern matching on records"
- Use the `age` type
- Use the title "Pattern Matching in Function Parameters" 
- Add an example with a tuple
- Update text accordingly